src/test/resources/spacestest.csv

```diff
@@ -0,0 +1,3 @@
+foo,bar,thing 
+07.81,7,81 
+25.42,7,42 
```

Comment:
CSV file from Windows echo -- there is an extra space (0x20) character at the end of each line that messes up the plot plugin.